### ISSUE TITLE
Error when looking at structure of table

### ISSUE THREAD

**Issue by alagoroVigo** [phpmyadmin/phpmyadmin#17270]
### Describe the bug

When pressing tab looking at the structure of a table, an error  goes:

Deprecation Notice in ./../../php/Twig/Extension/CoreExtension.php#1608
 array_key_exists(): Using array_key_exists() on objects is deprecated. Use isset() or property_exists() instead

### Server configuration

- Operating system: Ubuntu 18.04.6 LTS
- Web server: Apache 2.4.29
- Database version: 10.5.13-MariaDB-1:10.5.13+maria~bionic - mariadb.org binary distribution
                                 libmysql - mysqlnd 7.4.27
- PHP version: 7.24.27
- phpMyAdmin version:

Solution found searching Google: https://stackoverflow.com/questions/66222481/phpmyadmin-coreextension-php1608array-key-exists-using-array-key-exists-on

Basically the solution is changing line 1608 of file "/usr/share/php/Twig/Extension/CoreExtension.php"

where it says: `if (isset($object->$item) || \array_key_exists((string) $item, $object)) {`
if must say: `if (isset($object->$item) || property_exists($object, (string) $item)) {`
![Captura de pantalla de 2022-01-02 20-24-27](https://user-images.githubusercontent.com/25744405/147887250-5939d5cf-a74e-4257-b353-284fda9f7350.png)
![Captura de pantalla de 2022-01-02 20-30-13](https://user-images.githubusercontent.com/25744405/147887252-e32bbee1-0a2a-489d-97a1-d0212df6ac7c.png)

```
Deprecation Notice in ./../../php/Twig/Extension/CoreExtension.php#1608
 array_key_exists(): Using array_key_exists() on objects is deprecated. Use isset() or property_exists() instead

Backtrace

./../../../../../../var/lib/phpmyadmin/tmp/twig/de/dec2ede05e907b33280507b4d9d694ba8891607e0dc5aecfc824a064ccf4430e.php#187: twig_get_attribute(
,
,
,
string 'comment',
array,
string 'any',
boolean true,
boolean true,
boolean false,
integer 58,
)
Template.php#184: __TwigTemplate_5847c48934d38345c47414573809dc619be0359363c8df93e6ebf0d4cf6e05be->block_content(
array,
array,
)
./../../../../../../var/lib/phpmyadmin/tmp/twig/11/1183e916ad5f5a2c4dda969c27b0a576f932fa9f06be30d02d2efc068b0eeb9b.php#74: Twig\Template->displayBlock(
string 'content',
array,
array,
)
Template.php#395: __TwigTemplate_19c98bd5d8815beeecdd9e45571116ab1fc83076e7d9e3303f30c4a484eb2052->doDisplay(
array,
array,
)
Template.php#372: Twig\Template->displayWithErrorHandling(
array,
array,
)
./../../../../../../var/lib/phpmyadmin/tmp/twig/de/dec2ede05e907b33280507b4d9d694ba8891607e0dc5aecfc824a064ccf4430e.php#40: Twig\Template->display(
array,
array,
)
Template.php#395: __TwigTemplate_5847c48934d38345c47414573809dc619be0359363c8df93e6ebf0d4cf6e05be->doDisplay(
array,
array,
)
Template.php#372: Twig\Template->displayWithErrorHandling(
array,
array,
)
Template.php#380: Twig\Template->display(array)
TemplateWrapper.php#45: Twig\Template->render(
array,
array,
)
Template.php#132: Twig\TemplateWrapper->render(array)
StructureController.php#1565: PhpMyAdmin\Template->render(
string 'table/structure/display_structure',
array,
)
StructureController.php#171: PhpMyAdmin\Controllers\Table\StructureController->displayStructure(
array,
array,
,
array,
array,
boolean false,
)
Routing.php#187: PhpMyAdmin\Controllers\Table\StructureController->index(array)
index.php#18: PhpMyAdmin\Routing::callControllerForRoute(
string '/table/structure',
,
,
)
```

**Comment by williamdes** [comment]
Duplicate of #16746

**Comment by williamdes** [comment]
Hi @alagoroVigo 
This is now fixed, please have a look to #16746
You can try the latest 5.1 version in development [(phpMyAdmin 5.1+snapshot)](https://www.phpmyadmin.net/downloads/#snapshot_5.1+snapshot)
